sktime/tests/test_all_estimators.py

```diff
@@ -168,23 +165,26 @@ def test_estimator_tags(estimator_class):
 
 def test_inheritance(estimator_class):
     """Check that estimator inherits from BaseEstimator."""
-    Estimator = estimator_class
-
-    assert issubclass(Estimator, BaseEstimator), (
-        f"Estimator: {Estimator} " f"is not a sub-class of " f"BaseEstimator."
+    assert issubclass(estimator_class, BaseEstimator), (
+        f"Estimator: {estimator_class} " f"is not a sub-class of " f"BaseEstimator."
     )
 
     # Usually estimators inherit only from one BaseEstimator type, but in some cases
     # they may be predictor and transformer at the same time (e.g. pipelines)
-    n_base_types = sum(
-        [issubclass(Estimator, cls) for cls in VALID_ESTIMATOR_BASE_TYPES]
+    assert (
+        2
+        >= (
+            sum(
+                [issubclass(estimator_class, cls) for cls in VALID_ESTIMATOR_BASE_TYPES]
```

Comment:
I think this change makes the assertion more obscure to understand. Bracket overload...
Would you feel strongly about reverting it to its original state?

Reply:
This was autorefactored by an extension I use. I can revert back if it bother you.